src/views/AddForm.vue

```diff
@@ -0,0 +1,160 @@
+<script setup>
+import {ref, watch, computed, onMounted} from 'vue'
+import { RouterLink, useRouter} from 'vue-router'
+
+const personsArr = ref([])
+const fullName = ref('')
+const birthDate = ref('')
+const description = ref('')
+const editPerson = ref()
+const router = useRouter()
+
+ onMounted( () => {
+  if (localStorage.getItem('person')) {
+    editPerson.value = JSON.parse(localStorage.getItem('person'))
+    fullName.value = editPerson.value.firstName + ' ' + (editPerson.value.lastName ? editPerson.value.lastName : '') + ' ' +  (editPerson.value.middleName ? editPerson.value.middleName : '')
+    birthDate.value = editPerson.value.birthDate
+    description.value = editPerson.value.description
+  }
+})
+
+const addPersonData = () => {
+  const name = fullName.value.split(' ')
```

Comment:
При парсинге пользовательского ввода лучше обходить случайные двойные пробелы:
```suggestion
  const name = fullName.value.split(/\s+/);
```